File: PaintedSkin/src/main/java/org/alee/component/skin/exception/StrictModeException.java
package org.alee.component.skin.exception;

/**********************************************************
 *
 * @author: MY.Liu
 * @date: 2021/2/11
 * @description: xxxx
 *
 *********************************************************/
public final class StrictModeException extends RuntimeException {

    public StrictModeException(String message) {
        super(message);
    }

    public StrictModeException(Throwable throwable) {
        super(throwable);
    }
}

File: Library/PaintedSkin-Kotlin/src/main/java/org/alee/component/skin/exception/StrictModeException.kt
package org.alee.component.skin.exception

import org.alee.component.skin.util.INotProguard

/**
 * 摘要
 *
 * <p> 详细描述
 *
 * @author MingYu.Liu
 * created in 2023/8/31
 *
 */
internal class StrictModeException : RuntimeException, INotProguard {
    constructor(message: String) : super(message)
    constructor(throwable: Throwable) : super(throwable)
    constructor(message: String, throwable: Throwable) : super(message, throwable)
}
